src/math.cc

```diff
@@ -29,10 +33,12 @@ extern "C"
     return ::tan(x);
   }
 
+  #if defined(MODM_CMATH_GCC_VERSION) && (MODM_CMATH_GCC_VERSION < 110300L)
   float fabsf(float x)
   {
     return ::fabs(x);
   }
+  #endif
```

Comment:
I don't have an avr-gcc 11.3 at hand right now. What changes with regards to `fabsf` in the headers supplied by avr-gcc that we have to touch it? Is it a real function instead of a macro in 11.3?

Reply:
> What changes with regards to `fabsf` in the headers...

Good question. The three subroutines that I marked as _not_ _present_ in `math.cc` for the 11.3 compiler were (and need to be) marked that way since the designers of `<math.h>` elected to _inline` these functions at the C-header level by implementing them as `static` `inline` functions in the header itself. They can no longer be separate linkable entities in a C-file.

There is no way out of this one on the 11.3 port unless one would (as i do not recommend) modify `<math.h>` directly.

On the other hand, it would be conceptually thinkable to deliver a `math.h` with `avr-libstdcpp`, but I would not find that step necessary quite yet.